rules/rules.libsonnet

```diff
@@ -168,284 +168,15 @@
               ))
             ||| % $._config,
           },
-          {
-            // CPU utilisation is % CPU is not idle.
-            record: ':node_cpu_utilisation:avg1m',
-            expr: |||
-              1 - avg(rate(node_cpu_seconds_total{%(nodeExporterSelector)s,mode="idle"}[1m]))
-            ||| % $._config,
-          },
-          {
-            // CPU utilisation is % CPU is not idle.
-            record: 'node:node_cpu_utilisation:avg1m',
-            expr: |||
-              1 - avg by (node) (
-                rate(node_cpu_seconds_total{%(nodeExporterSelector)s,mode="idle"}[1m])
-              * on (namespace, %(podLabel)s) group_left(node)
-                node_namespace_pod:kube_pod_info:)
-            ||| % $._config,
-          },
-          {
-            // CPU utilisation per node, normalized by cluster-wide CPUs
-            record: 'node:cluster_cpu_utilisation:ratio',
-            expr: |||
-              node:node_cpu_utilisation:avg1m
-                *
-              node:node_num_cpu:sum
-                /
-              scalar(sum(node:node_num_cpu:sum))
-            ||| % $._config,
-          },
-          {
-            // CPU saturation is 1min avg run queue length / number of CPUs.
-            // Can go over 100%.  >100% is bad.
-            record: ':node_cpu_saturation_load1:',
-            expr: |||
-              sum(node_load1{%(nodeExporterSelector)s})
-              /
-              sum(node:node_num_cpu:sum)
-            ||| % $._config,
-          },
-          {
-            // CPU saturation is 1min avg run queue length / number of CPUs.
-            // Can go over 100%.  >100% is bad.
-            record: 'node:node_cpu_saturation_load1:',
-            expr: |||
-              sum by (node) (
-                node_load1{%(nodeExporterSelector)s}
-              * on (namespace, %(podLabel)s) group_left(node)
-                node_namespace_pod:kube_pod_info:
-              )
-              /
-              node:node_num_cpu:sum
-            ||| % $._config,
-          },
-          {
-            record: ':node_memory_utilisation:',
-            expr: |||
-              1 -
-              sum(node_memory_MemFree_bytes{%(nodeExporterSelector)s} + node_memory_Cached_bytes{%(nodeExporterSelector)s} + node_memory_Buffers_bytes{%(nodeExporterSelector)s})
-              /
-              sum(node_memory_MemTotal_bytes{%(nodeExporterSelector)s})
-            ||| % $._config,
-          },
           // Add separate rules for Free & Total, so we can aggregate across clusters
```

Comment:
Nit: This comment is out of date now as there is no "total" anymore. :o)

Reply:
fixed :)